Removed Duplicate

kulits opened the pull request:
Removed duplicate instance of "Magmar" in default config.

Commit:
Removed Duplicate

Removed duplicate instance of "Magmar" in default config.

DeadElle commented:
Nice catch ;)

DeadElle commented on PoGo.PokeMobBot.Logic/Settings.cs:
```diff
@@ -419,7 +419,6 @@ public class GlobalSettings
                 PokemonId.Scyther,
                 PokemonId.Magmar,
```
Nice catch, but following pokemon's IDs, the first one should be removed instead ;)

DurtyFree commented:
Please consider @DeadElle 's comment @idontcarebear

Commit:
Updated Pokemon Entries

Re-organized a few Pokemon by id and removed a duplicate occurrence of "Magmar" in the default config.

idontcarebear commented:
Oops, I missed that.  Here's an updated commit.

DurtyFree commented:
Thx:)
